pycbc/io/live.py

```diff
@@ -216,17 +226,22 @@ def __init__(self, ifos, coinc_results, **kwargs):
 
         # P astro calculation
         if 'padata' in kwargs:
-            padata = kwargs['padata']
+            assert len(coinc_ifos) < 3, \
+                f"p_astro can't handle {coinc_ifos} coinc ifos!"
```

Comment:
Same comment as above here, this fails for single-detector candidates?

Reply:
singles should have coinc_ifos of length 1 ?